.github/workflows/ci.yml

```diff
@@ -17,3 +17,10 @@ jobs:
       - run: yarn install
       - run: yarn build
       - run: yarn test --passWithNoTests
+  
+  integration:
+    runs-on: ubuntu-latest
+    permissions: {}
```

Comment:
This job should fail when an action using this template does anything more than checkout a repo (which works, incidentally, with no permissions, because it is a public repo).